---
title: "Integrative analysis identifies immune-related enhancers and lncRNAs perturbed by genetic variants associated with Alzheimer’s disease"
collection: posters
type: "Poster"
permalink: /posters/2017-05-23-IOA_INFERNO
venue: "UPenn Institute on Aging Retreat"
date: 2017-05-23
location: "Philadelphia, Pennsylvania"
---

[Poster PDF](/files/17.05.23.AAW_IOA.pdf). Received first place poster award in basic science category. 

Alexandre Amlie-Wolf, Mitchell Tang, Jessica King, Beth Dombroski, Yi-Fan Chou, Elizabeth Mlynarski, Gerard D. Schellenberg, Li-San Wang

We developed the INFERNO (INFERring the molecular mechanisms of NOncoding genetic variants) tool to analyze non-protein-coding genetic signals associated with late-onset Alzheimer’s disease (AD). We defined sets of variants in LD with any locus-wide significant variant and overlapped these expanded variant sets with enhancers from 112 FANTOM5 tissue facets and 127 Roadmap tissues and cell types and quantified their effects on transcription factor binding sites (TFBSs), revealing enhancer dysregulation in all 19 tag regions from IGAP. Using a unified tissue categorization to harmonize data sources identified a significant enrichment of enhancer overlaps in the blood/immune and connective tissue categories. To identify the affected target genes, we performed co-localization analysis of the GWAS signals with GTEx eQTL data across 44 tissues. This identified strongly co-localized eQTL signals in 15 tag regions, 9 of which contained variants overlapping enhancers from the same tissue class as the eQTL signal. In 6 of these 9 tag regions, we prioritized individual variants that disrupted or created TFBSs, and in 5 of them, we prioritized variants with high probabilities of individually underlying the co-localization signals. Both approaches identified a strong signal in the EPHA1 region targeting the EPHA1-AS1 long noncoding RNA (lncRNA) which was validated by luciferase assay. We identified similarly affected lncRNAs in several tag regions, and expression correlation showed that they regulated several aspects of the immune response, which has been previously implicated in AD pathogenesis. These results demonstrate the power of the principled integration of functional genomics data to characterize noncoding genetic signals. 
