---
title: "Inferring the Molecular Mechanisms of Noncoding AD Associated Genetic Variants"
collection: posters
type: "Poster"
permalink: /posters/2018-07-22-AAIC_INFERNO
venue: "Alzheimer's Association International Conference (AAIC)"
date: 2018-07-22
location: "Chicago, Illinois"
---

[Poster PDF](/files/18.07.22.AAW_AAIC.pdf)

Alexandre Amlie Wolf, Mitchell Tang, Beth A. Dombroski, PhD, Jessica Way, Nikolaos Vrettos, PhD, Yi Fan Chou, MS, Elizabeth E. Mlynarski, PhD, Christopher D. Brown, PhD, Gerard Schellenberg, PhD and Li San Wang, PhD

Background: Genome wide association studies (GWAS) have identified dozens of genetic variants associated
with Alzheimerâ€™s disease (AD). Most associations reside in the noncoding genome, where they affect tissue
specific enhancers. Furthermore, variants identified by GWAS may be proxies of truly causal variants in linkage
disequilibrium (LD). Methods: We applied our INFERNO algorithm to identify causal noncoding variants and their
regulatory effects. Significant variants from the top 19 loci from IGAP Stage I were expanded by LD. These
variants were tested for overlap with enhancer annotations across 239 tissues and cell types from FANTOM5 and
Roadmap and HOMER transcription factor binding sites (TFBSs) for 332 TFs. A Bayesian statistical model was
used to identify GWAS signals sharing co localized causal variants (posterior probability >= 0.5) with GTEx
expression quantitative trait loci (eQTL) across 44 tissues. GTEx RNA seq data were further used to predict
targets of long noncoding RNAs (lncRNAs). Tissue types from individual functional genomic data sources were
harmonized into 32 categories for unbiased identification of significantly enriched tissue contexts and variants
with concordant tissue specific functional support. Results: INFERNO identified 1,044 LD expanded variants
across all 19 loci, 852 of which affected enhancers and TFBSs, with enrichment of enhancer overlaps in the
immune related blood (p=0.0126) and the fibroblast containing connective tissue categories (p=0.0038).
INFERNO identified 154 co localized GWAS eQTL signals for 67 genes across 15 IGAP regions, including 59
signals overlapping concordant enhancers in matching tissue categories for 35 genes in 10 regions (ABCA7,
BIN1, CASS4, CD2AP, CD33, CELF1, CLU, EPHA1, INPP5D, ZCWPW1). Among these were 4 lncRNAs with
332 predicted targets enriched for dozens of immune response related pathways. Other regulatory signals
included effects on DNA damage repair and disruption of binding sites for 40 microRNAs. Allele specific enhancer
activity was validated by luciferase assays in K562 cells for strongly supported variants at the EPHA1, CD33, and
BIN1 loci. Conclusions: INFERNO was used to integrate GWAS signals with functional genomics data to infer
the perturbed genes, tissue contexts, and regulatory mechanisms affected by noncoding AD associated variants.
This provided novel post GWAS insights into the role of immunity in genetic susceptibility to AD and identified
potential therapeutic targets for future investigation.